olx_importer/management/commands/load_components.py

```diff
@@ -120,16 +121,21 @@ def create_content(self, static_local_path, now, component_version):
 
         hash_digest = create_hash_digest(data_bytes)
 
-        content, _created = Content.objects.get_or_create(
+        content, created = Content.objects.get_or_create(
             learning_package=self.learning_package,
             mime_type=mime_type,
             hash_digest=hash_digest,
             defaults = dict(
-                data=data_bytes,
                 size=len(data_bytes),
                 created=now,
             )
         )
+        if created:
+            content.file.save(
+                f"{content.learning_package.uuid}/{hash_digest}",
+                ContentFile(data_bytes),
+            )
```

Comment:
Is this saving _every_ Content's data to both MySQL and S3 ?

Reply:
No. This code is really hacky. The only content that gets here are the static assets referenced from XBlocks, and they are stored only in storages/s3 (the `data=data_bytes` line was removed, so no db-level storage).

There's a separate part where the XBlock's own OLX data is read in, and that part remains db-only for storage.

Reply:
Ah ok, I was reading this quickly and didn't catch that. Makes sense.